tests/unit/data_formats/datumaro/test_datumaro_format.py

```diff
@@ -64,7 +64,7 @@ def _test_save_and_load(
         [
             pytest.param(
                 "fxt_test_datumaro_format_dataset",
-                compare_datasets_strict,
+                compare_datasets,
```

Comment:
To replace `compare_datasets_strict`, we need `require_media=True` argument for `compare_datasets(a, b, require_media=True)`. Without this flag, `compare_datasets` do not check equivalence between the media entities.

Reply:
Thank you for letting me know this. But below True is for require_media as a compare keyword argument.

Reply:
I got it, but L206 still needs it.